### PR TITLE
moved calls with promises to it's own file

### PR DESCRIPTION
**Pull request by michaelmld**

### CODE REVIEW

File: lib/lolAPI.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 /**
+ * Created by michael.dang on 4/26/17.
+ */
+/**
```
Comment (saxire) [inline lib/lolAPI.js]: Uh 2 created by comment blocks?

File: lib/summonerComparer.js [hunk 2]
```diff
@@ -5,7 +5,7 @@
 var championMap = require('../public/static/champions.json');
 
 //Takes in a list of tuples of Objects that contain the id of a champion and the summoner's stats for the particular champion
-exports.compareCommonChampions = function (commonChampions) {
+exports.compareCommonChampions = (commonChampions) => {
```
Comment (saxire) [inline lib/summonerComparer.js]: Can you explain this to me in high level terms?

File: lib/lolAPI.js (hunk 1 above)
Comment (michaelmld) [inline lib/lolAPI.js]: lol

**Comment by saxire** [comment]
Everything else looks fine.

File: lib/summonerComparer.js [hunk 3]
```diff
@@ -136,11 +135,11 @@ var compareAssists = function(summonerAStats, summonerBStats) {
     return createResultObject(summonerAAvgAssists, summonerBAvgAssists, "Assists", "some reason", winner)
 };
 
-var getWinnerOffset = function(offset) {
+var getWinnerOffset = (offset) => {
     return (offset > 0 ? 1 : (offset < 0 ? -1 : 0));
 }
 
-var createResultObject = function(summonerStatA, summonerStatB, category, reason, winner) {
+var createResultObject = (summonerStatA, summonerStatB, category, reason, winner) => {
```
Comment (saxire) [inline lib/summonerComparer.js]: comment turn off test

File: lib/summonerComparer.js (hunk 2 above)
Comment (michaelmld) [inline lib/summonerComparer.js]: > An arrow function expression has a shorter syntax than a function expression and does not bind its own this, arguments, super, or new.target. These function expressions are best suited for non-method functions, and they cannot be used as constructors.

Just making use of ES6's arrow functions

**Pull request merged**